src/DependencyInjection/DI.External.Tests/test/Microsoft.Extensions.DependencyInjection.ExternalContainers.Tests.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>$(DefaultNetCoreTargetFramework);net472</TargetFrameworks>
     <RootNamespace>Microsoft.Extensions.DependencyInjection</RootNamespace>
     <SignAssembly>false</SignAssembly>
+    <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
```

Comment:
Prefer `%(AllowExplicitReference)` metadata on the specific reference if that's supported in this repo

Reply:
Done